### PR TITLE
base routing logic, added home and play components, bootstrap stuff

### PR DESCRIPTION
**Pull request by daryltanwk**

### CODE REVIEW

File: frontend/src/app/app.module.ts [hunk 1]
```diff
@@ -5,10 +5,14 @@ import { HttpModule } from '@angular/http';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { HomeComponent } from "app/home/home.component";
```
Comment (soedar) [inline frontend/src/app/app.module.ts]: maybe prefix ./ to be more consistent? i dont know

also singe quote string please.

File: frontend/src/app/play/play.component.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  <div class="row">
+    <div class="col-xs-12">
+      <h1>{{placeholder}}</h1>
+      <br>
+      <h3><small>yes, it is.</small></h3>
+      <br>
+      <h1>{{placeholder2}}</h1>
+    </div>
+  </div>
+</div>
```
Comment (soedar) [inline frontend/src/app/play/play.component.html]: https://robots.thoughtbot.com/no-newline-at-end-of-file

File: frontend/src/app/app.module.ts [hunk 3]
```diff
@@ -5,10 +5,14 @@ import { HttpModule } from '@angular/http';
...
+import { PlayComponent } from './play/play.component';
 
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
+    HomeComponent,
+    PlayComponent
```
Comment (soedar) [inline frontend/src/app/app.module.ts]: what do you think about adding a trailing comma at the end of list?

**Comment by xuatz** [comment]
might want to consider https://github.com/prettier/prettier to abstract away this problem

**Comment by soedar** [comment]
that's really interesting project, we can try that.

File: frontend/src/app/app.module.ts (hunk 1 above)
Comment (daryltanwk) [inline frontend/src/app/app.module.ts]: omg sabo-ed by auto-imports.

File: frontend/src/app/app.module.ts (hunk 3 above)
Comment (daryltanwk) [inline frontend/src/app/app.module.ts]: i remember this caused me syntax error once upon a time... lol

File: frontend/src/app/app-routing.module.ts [hunk 4]
```diff
@@ -1,9 +1,17 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
+import { HomeComponent } from "app/home/home.component";
```
Comment (soedar) [inline frontend/src/app/app-routing.module.ts]: single quote pls

**Pull request merged**